app/femr/data/models/mysql/Patient.java

```diff
@@ -53,6 +53,9 @@
     private List<PatientEncounter> patientEncounters;
     @Column(name = "isDeleted", nullable = true)
     private DateTime isDeleted;
+    @Column(name = "deleted_by_user_id", unique = false, nullable = false) 
```

Comment:
nullable should be true because the column always has a null value unless a patient has been deleted